### ISSUE TITLE
escape char in `by` not work

### ISSUE THREAD

**Issue by n4mine** [VictoriaMetrics/VictoriaMetrics#743]
**Describe the bug**
`sum(ooo) by (k\|v)` is not work. `k|v` is a label key.

**To Reproduce**
```
while : ; do v=$(date +%s) ; echo $v ; curl -d "ooo,k=$(shuf -i 1-5 | head -1),k|v=$(shuf -i 1-5 | head -1) v=${v} ${v}000000000" -X POST 'http://127.0.0.1:8428/write'  ; sleep 1 ; done
```

### escape label key works fine
![333](https://user-images.githubusercontent.com/3345153/92096629-b3fc6380-ee09-11ea-869e-a41c932789b1.jpg)

### `by` non escape k works fine
![111](https://user-images.githubusercontent.com/3345153/92095998-fb362480-ee08-11ea-8974-926aac9228aa.jpg)

### `by` escape k not work
![222](https://user-images.githubusercontent.com/3345153/92096129-1c971080-ee09-11ea-8db5-f30e00de9749.jpg)

**Expected behavior**
`sum(ooo) by (k\|v)` works as `sum(ooo) by (k)`

**Version**
The line returned when passing `--version` command line flag to binary. For example:
```
./bin/victoria-metrics -version
victoria-metrics-20200903-090605-heads-master-0-g764b3d4f
```

**Comment by n4mine** [comment]
thank you  @valyala 

it's fixed.

**Comment by valyala** [comment]
The bugfix will be included in [v1.40.1](https://github.com/VictoriaMetrics/VictoriaMetrics/releases).
